public/scripts/search-results.js

```diff
@@ -5,6 +5,11 @@ const controller = new ResultsController(template);
 
 controller.onResultsAdded.addEventListener(function(){
     document.querySelector("#resultCount").innerText = (controller.getStart()+1) + "-" + controller.getCount();
+    if (controller.getCount() >= controller.getTotal()) {   
+        Array.prototype.forEach.call(document.getElementsByClassName("load-more-results"), function(element) {
+            element.className += " invisible";
```

Comment:
Feel like I'm nitpicking at a bit here but 
```suggestion
            element.classList.add("invisible");
```
might be a bit cleaner here - avoids logic errors resulting in the class added twice, or us having to worry about spacing. :smile:

Reply:
Thanks! Yes, much better! And we can then use element.classList.remove too if needed.

Reply:
Nitpicking is good!